external/kafka-0-10-sql/src/test/scala/org/apache/spark/sql/kafka010/KafkaDataConsumerSuite.scala

```diff
@@ -235,7 +235,7 @@ class KafkaDataConsumerSuite
     try {
       val futures = (1 to numConsumerUsages).map { i =>
```

Comment:
If you're going to modify this class,  then `s/i/_/`.

Reply:
Removed.